xaos.appdata.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.xaos_project.XaoS</id>
```

Comment:
but why is this here since you have a patch for upstream's?

Reply:
They changed it so many times since last release that it is inconvenient to patch.

Reply:
also pro-tip: since you get the upstream source from git, just checkout the git tree, change the files in the tree, and then `git diff`. (or commit and then `git show`). Redirect that into the patch file.